aws-marketplace-install.md

```diff
@@ -24,34 +26,55 @@ Please contact us at support@kubecost.com with any questions and we'd be happy t
 }
 ```
 
-## Step 2. Create an IAM role with the appropriate trust relationships
-We recommend doing this via eksctl. More detail and how to set up the appropriate trust relationships is available [here](https://docs.aws.amazon.com/eks/latest/userguide/create-service-account-iam-policy-and-role.html).
+## Step 2. Create an IAM role for service account (IRSA).
+We recommend doing this via [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html). The command below helps to automate these manual steps:
+- Create an IAM role with AWS managed IAM policy.
+- Create a K8s service account name `awsstore-serviceaccount` in your Amazon EKS cluster.
+- Set up trust relationship between the created IAM role with awsstore-serviceaccount.
+- Modify `awsstore-serviceaccount` annotation to associate it with the created IAM role 
+
+Please remember to replace `CLUSTER_NAME` with your actual Amazon EKS cluster name.
+
 ```
 eksctl create iamserviceaccount \
     --name awsstore-serviceaccount \
     --namespace kubecost \
-    --cluster cluster_name \
-    --attach-policy-arn IAM_policy_ARN \
+    --cluster CLUSTER_NAME \
+    --attach-policy-arn arn:aws:iam::aws:policy/AWSMarketplaceMeteringRegisterUsage \
     --approve \
     --override-existing-serviceaccounts
 ```
+For more detail and how to set up the appropriate trust relationships is available [here](https://docs.aws.amazon.com/eks/latest/userguide/create-service-account-iam-policy-and-role.html).
 
+> Note: Your Amazon EKS cluster need to have IAM OIDC provider enabled to set up IRSA. You can learn more on how to enable IAM OIDC provider at this [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) 
 ## Step 3. Deploy Kubecost with attached IAM role
 
-Access Helm install steps available at [kubecost.com/install](http://kubecost.com/install). 
+- Define which available version you would like to install using this following command (you can check available version title from the AWS Marketplace product):
+
+`export IMAGETAG=prod-1.94.2`
 
-Supply the following parameters to your _helm install_ command.
+- Deploy Kubecost with `Helm` using the following command:
 
 ```
---set kubecostProductConfigs.productKey.enabled=true \
---set kubecostProductConfigs.productKey.key="replace-with-product-key" \
---set prometheus.alertmanager.enabled=false \
---set prometheus.nodeExporter.enabled=false \
---set global.grafana.enabled=false \
---set global.grafana.proxy=false \
---set awsstore.useAwsStore=true \
---set awsstore.imageNameAndVersion=gcr.io/kubecost1/awsstore:latest
+helm upgrade -i kubecost kubecost/cost-analyzer \
+    --namespace kubecost --create-namespace \
+    --set prometheus.nodeExporter.enabled=false \
+    --set global.grafana.enabled=false \
+    --set global.grafana.proxy=false \
+    --set awsstore.useAwsStore=true \
```

Comment:
We need to attach a service account to the aws pod via annotation right? Otherwise the IAM role won't be attached? Or is that somehow already done?

Reply:
Hi @AjayTripathy , it has already been done at Step 2 with this following command which automatically create the service account, IAM role and add the annotation to the service account:

eksctl create iamserviceaccount \
    --name awsstore-serviceaccount \
    --namespace kubecost \
    --cluster CLUSTER_NAME \
    --attach-policy-arn arn:aws:iam::aws:policy/AWSMarketplaceMeteringRegisterUsage \
    --approve \
    --override-existing-serviceaccounts